### PR TITLE
Add two empty lines before a method

### PR DESCRIPTION
**Pull request by RajkumarVelusamy**
Hi Alex,
This merge request has add 2 empty lines before a method rule and Fix for that.
In addition to that, there will be a warning if there are more than 2 empty line before a method,

### CODE REVIEW

File: src/parser/parser.ts [hunk 1]
```diff
@@ -97,6 +97,11 @@ export interface Method extends Member {
 
 	statements: Statement[];
 
+	/**
+	 * One line before the method seperator
+	 */
+	oneLineB4: number
```
Comment (kewtree1408) [inline src/parser/parser.ts]: At first I thought it is special meaning of B4 somewhere.. But now I think it's like the name: B4 - before. Is it correct? It was not obvious at first look :(

File: CHANGELOG.md [hunk 2]
```diff
@@ -40,6 +40,7 @@ These are the current rules:
 - MethodSeparator
 - MethodParametersOnNewLine
 - PropertyLiteralCase
+- TwoEmptyLines above a method
```
Comment (kewtree1408) [inline CHANGELOG.md]: Do we need to add this new feature here? Or we need to add this line to a new version like # v1.4.2?

File: __tests__/methodDoc-test.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	let separatorDiagnostics: api.Diagnostic[] = [];
+
+	beforeAll(async () => {
+		docDiagnostics = await utils.getDiagnostics(procName, MethodDocumentation.name);
+		emptyLineDiagnostics = await utils.getDiagnostics(procName, TwoEmptyLines.name);
+		separatorDiagnostics = await utils.getDiagnostics(procName, MethodSeparator.name);
+	});
+
+	test('allProblemsNoLineAbove', () => {
+		// expect(messageOnLine(0, docDiagnostics)).toBe('TODO');
```
Comment (kewtree1408) [inline __tests__/methodDoc-test.ts]: Do you still need these comments?

File: CHANGELOG.md (hunk 2 above)
Comment (kewtree1408) [reply CHANGELOG.md]: Will it be better to add about new rule into `pslLint/cli/README.md`? What do you think? )

File: src/parser/parser.ts (hunk 1 above)
Comment (atiplea) [reply src/parser/parser.ts]: I agree the name is confusing. I also think this may not be valuable as a property.

File: CHANGELOG.md (hunk 2 above)
Comment (atiplea) [reply CHANGELOG.md]: Yes, that readme shows up here: https://www.npmjs.com/package/psl-lint

We should keep that one up-to-date.

File: __tests__/methodDoc-test.ts (hunk 3 above)
Comment (atiplea) [reply __tests__/methodDoc-test.ts]: They're placeholders to show how to write the tests. They will be uncommented and updated soon.

**Pull request merged**